src/main/java/org/reactome/addlinks/fileprocessors/VGNCFileProcessor.java

```diff
@@ -0,0 +1,62 @@
+package org.reactome.addlinks.fileprocessors;
+
+import org.reactome.addlinks.EnsemblBioMartUtil;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+public class VGNCFileProcessor extends FileProcessor{
+
+    public VGNCFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public VGNCFileProcessor()
+    {
+        super();
+    }
+
+    private static final int vgncIdentifierIndex = 1;
+    private static final int uniprotIdentifiersIndex = 21;
+
+    /**
+     * Build map of UniProt identifiers to Vertebrate Gene Nomenclature Committee (VGNC) identifiers that is used to create VGNC cross-references in database.
+     * @return - Map<String, List<String>
+     */
+    @Override
+    public Map<String, List<String>> getIdMappingsFromFile()
+    {
+        Map<String, List<String>> mappings = new HashMap<>();
+        try {
+            this.unzipFile(this.pathToFile, true);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        Path inputFilePath = Paths.get(this.pathToFile.toAbsolutePath().toString().replace(".gz", ""));
+
+        List<String> lines = new ArrayList<>();
+        try {
+            lines = EnsemblBioMartUtil.getLinesFromFile(inputFilePath, true);
```

Comment:
This might be clearer to have a method in `EnsemblBioMartUtil` called `getFileLinesWithoutHeader` that delegates to `getLinesFromFile`.  I try to avoid passing in raw booleans as arguments since it's hard to tell what is being marked as true or false without looking up the method implementation and breaking the flow of reading the code.

```
private List<String> getFilesLinesWithoutHeader(Path filePath) {
    boolean skipHeader = true;
    return getLinesFromFile(filePath, skipHeader);
}
```